### PR TITLE
feat: Use video tile for description, if description is empty

### PR DESCRIPTION
**Pull request by lanmengyujing**
# Reason for this change
See detail in https://github.com/brightcove/videojs-schema/issues/23

### CODE REVIEW

File: src/plugin.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const schema = function(options) {
     if (options.preferLongDescription) {
       ld.description = this.mediainfo.longDescription || this.mediainfo.description;
```
Comment (mister-ben) [inline src/plugin.js]: This should also fall back to name.

**Review by mister-ben — CHANGES_REQUESTED** [review]
This makes sense if it produces valid schema when no description is set, notwithstanding that having separate values may be better for SEO.

File: src/plugin.js (hunk 1 above)
Comment (lanmengyujing) [reply src/plugin.js]: thanks for the suggestion

**Comment by mister-ben** [comment]
Thanks!

**Comment by lanmengyujing** [comment]
@mister-ben thanks for help merge the pull request. How can I get the latest code for this plugin?
Can I use this https://cdn.jsdelivr.net/npm/videojs-schema/dist/videojs-schema.min.js?

**Comment by mister-ben** [comment]
I just published a new version to NPM, jsdelivr etc will return that soon although I'm not sure how long they cache. For me https://cdn.jsdelivr.net/npm/videojs-schema@latest/dist/videojs-schema.min.js is already returning v0.5.0.

**Comment by lanmengyujing** [comment]
@mister-ben thanks for helping publish the new version to NPM. How can I use this library using ES6 module **import** with https://github.com/brightcove/react-player-loader ?
I did not find any documentation about it.  Appreciate it if there is a demo or any documentation I can refer to.

**Comment by mister-ben** [comment]
With react player loader, you may need to register the plugin on the `window.videojs` created by the player script, on the player success callback

```js
import window from "global/window";
import schema from "videojs-schema";
```

```js
const onSuccess = function (success) {
  const player = success.ref;
  if (!player.hasPlugin("schema")) {
    window.videojs.registerPlugin("schema", schema);
  }
  player.schema();
};
```

```js
ReactDOM.render( <BrightcovePlayer onSuccess={onSuccess} ...
```

Adding the script to the player config is recommended instead.

**Comment by lanmengyujing** [comment]
@mister-ben Thanks for sharing the code, that really helps.
Why the recommended way is adding the schema script to the player config?  Are there any drawbacks of using this video schema plugin with ES import?

**Pull request merged**